Automattic/newspack-theme
# Reorder single post

### All Submissions:


### Changes proposed in this Pull Request:

This PR is a version rescuing changes I made, and tried to recover, from #69. 

This PR makes updates to the single article's appearance and order to match the mockups; specific changes include:

1. Moving Jetpack's sharing icon output to site up next to the title.
2. Moving the category output above the title.
3. Adjusts general styles and spacing.

### How to test the changes in this Pull Request:

1. Apply the PR.
2. Activate Jetpack's share buttons; set to just show icons.
3. Review single stories and compare against the mockups; make sure they generally look correct. Note: the sharing icons display different with AMP enabled (large and rectangular, not smaller and rounded) - I'm less familiar with the styling there, so have opted not to try to override it for now.

### Other information:
